src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name, health = 100, hidden = false) {
+    this.name = name;
+    this.health = health;
+    this.hidden = hidden;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health = 100, hidden = false) {
```

Comment:
There is no need to redeclare the default value for 'health' and 'hidden' in the Herbivore constructor since these are already handled by the Animal class constructor. You can remove the parameters from the Herbivore constructor and rely on the superclass constructor to set the default values.